utility/module.go

```diff
@@ -60,3 +139,39 @@ func (u *utilityModule) GetModuleName() string {
 func (u *utilityModule) GetMempool() mempool.TXMempool {
 	return u.mempool
 }
+
+func (u *utilityModule) GetLogger() *modules.Logger {
+	return u.logger
+}
+
+func (u *utilityModule) GetActorModules() []modules.Module {
+	return u.actorModules
+}
+
+func (u *utilityModule) GetServicerModule() modules.Module {
+	return u.svc
+}
+
+// validateActorModuleExclusivity validates that the actor modules are enabled in a valid combination.
+// TODO: There are probably more rules that need to be added here.
+func validateActorModuleExclusivity(m *utilityModule, cfg *configs.Config) error {
```

Comment:
DItto above.

1. Do not pass in `utilityModule`, make it a receiver
2. Config is already part of the `utilityModule` so access that directly

Reply:
We need the root config for this bit, and didn't want to add that as part of the module.